packages/phrases/src/locales/en.ts

```diff
@@ -281,6 +281,7 @@ const translation = {
       save_success: 'Saved!',
       send: 'Send',
       send_error_invalid_format: 'Invalid input',
+      edit_config_label: 'Enter your json here',
```

Comment:
nit: better align with the zh_CN version? `JSON`

Reply:
It's aligned with the design and will be updated before V1 launched by @fleuraly .